src/pages/Admin/BackgroundSelect/BackgroundSelect.scss

```diff
@@ -11,25 +11,35 @@
   background-color: $dark;
   background-position: center;
 
+  &__maps-header {
+    font-size: $font-size--md;
+    font-weight: normal;
+    color: rgba(255, 255, 255, 0.5);
+    margin: $spacing--xxl auto $spacing--md auto;
+  }
+
   &__map-grid {
-    display: grid;
-    width: 50%;
-    grid-gap: $spacing--md;
-    grid-template-columns: repeat(3, 1fr);
+    display: flex;
+    flex-direction: row;
+    justify-content: space-evenly;
+    flex-wrap: wrap;
+    width: 100%;
   }
 
   &__map {
-    width: 100%;
-    padding-bottom: 50px;
+    width: 23%;
+    padding-bottom: 60px;
```

Comment:
Move this at the top of the file and reuse it.

Reply:
fixed in 21dc9b1e0834b6219b5ccf8a32fca56a6a2484c6